setup.py

```diff
@@ -21,17 +23,18 @@ def setup_package():
 
     _groups_files = {
         'base': 'requirements.txt', #basic requirements
+        'pip': 'requirements_pip.txt', #requirements with pip
         'tests': 'requirements_tests.txt', #requirements for tests
         'docs': 'requirements_docs.txt' #requirements for building docs
     }
     reqs = _get_requirements_from_files(_groups_files)
     install_reqs = reqs.pop('base')
     extras_reqs = reqs
 
-    setup(name='spopt', #name of package
+    setup(name=package,
           version=__version__,
           description='', #short <80chr description
-          url='', #github repo
+          url='https://github.com/pysal/'+package, #github repo
           maintainer='',
           maintainer_email='',
           test_suite = 'nose.collector',
```

Comment:
i dont know that it matters but we're using pytest rather than nose

Reply:
or, we'd intended to anyway. Not sure an official decision has been made since we dont have any tests yet

Reply:
it probably does matter... i'll make the switch.

Reply:
@knaaptime would this simply be `test_suite = 'pytest'`?